src/pathfora.js

```diff
@@ -611,8 +611,9 @@
       if (!total) {
         totalImpressions = 1;
       } else {
-        parts = total.split(","),
+        parts = total.split("_"),
         totalImpressions = parseInt(parts[0]) + 1;
+        parts = parts.length === 1 ? total.split(",") : parts;
```

Comment:
probably worth adding a comment here (and all places that do the same conversion) that notes the date we should end support for it. no reason to keep this around after all the pre-existing cookies expire.